src/PowerShell/Microsoft.WinGet.Client/Commands/Common/BaseInstallCommand.cs

```diff
@@ -57,7 +57,13 @@ public string Location
         /// Gets or sets a value indicating whether to skip the installer hash validation check.
         /// </summary>
         [Parameter(ValueFromPipelineByPropertyName = true)]
-        public SwitchParameter AllowHashMismatch { get; set; }
+        public SwitchParameter AllowHashMismatch { get; set; }
+
+        /// <summary>
```

Comment:
We do not want to expose this in Powershell module yet, so please revert changes to this file